homeassistant/components/alexa/capabilities.py

```diff
@@ -994,6 +999,29 @@ def capability_resources(self):
             self._resource.add_mode(f"{cover.ATTR_POSITION}.custom", ["Custom"])
             return self._resource.serialize_capability_resources()
 
+        if self.instance == f"{light.DOMAIN}.{light.ATTR_EFFECT}":
+            self._resource = AlexaModeResource(["Effect"], False)
+
+            effect_list = self.entity.attributes.get(light.ATTR_EFFECT_LIST, 0)
+            if effect_list:
+                for effect in effect_list:
+                    self._resource.add_mode(f"{effect}", [f"{effect}"])
```

Comment:
```suggestion
                    self._resource.add_mode(f"{light.ATTR_EFFECT}.{effect}", [f"{effect}"])
```
Include the Mode identifier.